pkg/eth/trace.go

```diff
@@ -81,80 +94,33 @@ func (c *TraceClient) DebugTraceCall(
 		blockStr = hexutil.EncodeBig(block)
 	}
 
-	payload := map[string]any{
-		"method":  "debug_traceCall",
-		"id":      1,
-		"jsonrpc": "2.0",
-		"params": []any{
-			paramData,
-			blockStr,
-			map[string]any{
-				"tracer": "callTracer",
-				"tracerConfig": map[string]interface{}{
-					"onlyTopCall": false,
-					"withLog":     true,
-				},
+	var callResp json.RawMessage
+	if err := c.rpcClient.CallContext(ctx, &callResp,
```

Comment:
Same as above

Reply:
updated in https://github.com/KyberNetwork/tradinglib/pull/65/commits/be362697cfebc26dcf947540e92dc89402614698